ddtrace/internal/gitmetadata.py

```diff
@@ -0,0 +1,128 @@
+import typing
+
+from envier import Env
+
+from ddtrace.internal.utils import formats
+
+
+_GITMETADATA_TAGS = None  # type: typing.Optional[typing.Dict[str, str]]
+
+TAG_REPOSITORY_URL = "git.repository_url"
+TAG_COMMIT_SHA = "git.commit.sha"
+
+TRACE_TAG_REPOSITORY_URL = "_dd.git.repository_url"
+TRACE_TAG_COMMIT_SHA = "_dd.git.commit.sha"
```

Comment:
If we plan to import these constants in other modules then we should move them to `ddtrace/ext/git.py`

Reply:
We have no plans use it outside of this module (should I rename it with "_")?

Reply:
Sure but that's not necessary since this is module exists in the internal package.